src/absil/il.fsi

```diff
@@ -1144,7 +1143,7 @@ type ILOverridesSpec =
     member MethodRef: ILMethodRef
     member EnclosingType: ILType 
 
-// REVIEW: fold this into ILMethodDef
+// REVIEW: fold this into ILMethodDef.
```

Comment:
Can we add an issue to do this work?, then add the github issue number.  If we close the issue without doing the work, the issue should note that these comments should be updated.

How does that sound?